templates/firstaide/edit_post.html

```diff
@@ -3,16 +3,16 @@
 {% block card_content %}
 <br>
 <center>
-    <h2> 
-        <font color="#2c0f00"> FirstAide APIs : Posts</font> 
+    <h2>
+        <font color="#2c0f00"> FirstAide APIs : Posts</font>
     </h2>
 </center>
 <div class="well">
     <h2 class="text-center inverted-font">Edit Card</h2>
     <br>
     <form class="form-horizontal" method="post">
         {% csrf_token %}
-        
+
```

Comment:
Same as above. Please make sure you don't commit auto-formatted changes next time.